uart/pong.toit

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2021 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the EXAMPLES_LICENSE file.
+
+import uart
+import writer show Writer
+import gpio
+
+RX ::= 33
+TX ::= 32
+
+main:
+  // The UART port can be configured in many ways.
+  // See https://libs.toit.io/uart/class-Port#constructor(10%2C0%2C0%2Cbaud_rate%2Ccts%2Cdata_bits%2Cinvert_rx%2Cinvert_tx%2Cparity%2Crts%2Crx%2Cstop_bits%2Ctx)
```

Comment:
Any chance we can use a link that doesn't include the signature? This looks very fragile to API extentions.

Reply:
Currently it's not possible.
Linking just to the 'Port' class.